contributing/code/bugs.rst

```diff
@@ -32,9 +33,19 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
+* If you want to provide a stack trace you got on an html page, be sure to
```

Comment:
I'm also for plain text, because it allows search engines to match exception messages that people encounter, so they may be redirected to the correct issue just by googling the exception itself